.github/workflows/ci.yml

```diff
@@ -37,3 +37,24 @@ jobs:
       - name: Run tests
         run: |
           pytest -v
+
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ruff
+
+      - name: Run Ruff
+        id: ruff
+        run: |
+          # It would be nice to push the changes back to the PR branch, but that's difficult to do with forks
+          ruff format --check
```

Comment:
I would suggest to run the following command here:
```
ruff check --select I --fix
ruff format
```
This will also sort the imports :)

https://docs.astral.sh/ruff/formatter/#sorting-imports